src/applications/coronavirus-research/config/form.js

```diff
@@ -1,29 +1,31 @@
 import environment from 'platform/utilities/environment';
-import fullSchema from 'vets-json-schema/dist/COVID-VACCINE-TRIAL-schema.json';
+// import fullSchema from 'vets-json-schema/dist/COVID-VACCINE-TRIAL-schema.json';
+import fullSchema from './temp.json';
```

Comment:
@stephenBarrs this leftover scaffolding from your local work?

Reply:
ah i see the temp file down below. guessing this is to handle the chicken/egg dilemma with vets-json-schema...?

Reply:
yep this will need a corresponding PR in `vets-json-schema`  I wanted to see this in review instance so pushed out PR